server/src/main/resources/file.conf

```diff
@@ -118,6 +118,8 @@ transaction {
   #schedule delete expired undo_log in milliseconds
   undo.log.delete.period = 86400000
   undo.log.table = "undo_log"
+  # sequence name, optional, default is table name + "_SEQ", case insensitive
+  undo.log.sequence = "undo_log_seq"
```

Comment:
undo_log table always create in client side.

Reply:
whatever the undo_log and its related sequence configuration and description write one place?